static/templates/invitation_failed_error.hbs

```diff
@@ -4,3 +4,10 @@
     <li>{{this}}</li>
     {{/each}}
 </ul>
+{{#if is_invitee_deactivated}}
+    {{#if check_if_admin}}
+    <p>{{t "You can reactivate deactivated users from "}}<a href="#organization/deactivated-users-admin">{{t "organization settings."}}</a></p>
+    {{else}}
+    <p>{{t "Organization administrators can reactivate deactivated users." }}</p>
+    {{/if}}
+{{/if}}
```

Comment:
Can you add a test for this template in `frontend_tests/node_tests/templates.js`?

Reply:
@hackerkid not really sure how to go about this ...haven't worked with frontend tests before any resource I can look into for this? I checked the docs but didn't find anything related to writing node tests.

Reply:
@Tlazypanda Go through the other tests in templates.js. And create a simple test for `invitation_failed_error.hbs `.

Reply:
@hackerkid @timabbott Added node test for `invitation_failed_error.hbs`